src/api/apiProducts.ts

```diff
@@ -0,0 +1,19 @@
+import apiEndpoints from "@/api.endpoints";
+
+import handleErrors from "@/utils/handleErrors";
+import { Game } from "@/components/gameCard/gameCard";
+import * as api from "./requests";
```

Comment:
Recommendation; such import will import everything, but here required only 1..3 endpoints. It will be good for optimization import only required items